woocommerce-services.php

```diff
@@ -341,6 +341,8 @@ public function load_textdomain() {
 		 * @codeCoverageIgnore
 		 */
 		public function init() {
+			add_action( 'admin_init', array( $this, 'admin_enqueue_scripts' ) );
```

Comment:
`admin_enqueue_scripts` only `registers` the bundles, doesn't do any actual work, so it's safe to execute always.